t/18-qemu.t

```diff
@@ -6,14 +6,19 @@ use warnings;
 
 use Test::More;
 use Test::Warnings 'warnings';
-use Mojo::File qw(tempfile path);
+use Mojo::File qw(tempfile tempdir path);
```

Comment:
I can't see that `tempdir` is actually used. I'm also wondering why we can't make more use of it. It had the advantage that the source tree isn't polluted at all and there would be no potential problems when running tests concurrently. It looks like you've tested using `tempdir` but it didn't work for some reason.

Reply:
Oh right. Yeah, that is a leftover. `tempdir` requires the dir to end with `XXX`, so it's not possible to use it with a specific directory.

Reply:
*That* is the reason? What prevents you from creating a directory `testresults` within the `tempdir`?

Reply:
Simply because I don't know how to influence the location of the directory. If you look into `t/18-qemu.t` you will only see that I create that directory and at the end remove it. But the test code actually fills it.

Maybe it will just work if I do a `chdir` to the tempdir. I have to try that out.

Reply:
No, `chdir` doesn't wor. Then it cannot find other data:
`Could not open 'data/Core-7.2.iso'`

Reply:
It works if I specify `"$Bin/data/...` everywhere

Reply:
We can of course just go for what works for now because out of my head I also don't know how to influence that directory. Maybe it is possible via some environment variable or `$bmwqemu::vars`.

Reply:
I'll try if I also can get it working with the other tests.
Being able to run tests in parallel would be cool.

Reply:
@Martchus  I changed everything to use `tempdir` now

Reply:
Nice, so `chdir` works after all but causes some additional effort because other parts were relying on the previous working directory.